ScratchDjango/User/views.py

```diff
@@ -42,8 +53,16 @@ def register_user(request):
 def login_user(request):
     user = authenticate(email=request.data["email"], password=request.data["password"])
     if user:
-        token = get_refresh_token(user)
-        return Response(token, status=status.HTTP_200_OK)
+        if not user.otp_enabled:
+            token = get_refresh_token(user)
+            return Response({"status": token}, status=status.HTTP_200_OK)
```

Comment:
Token values variables should be with the correct name. Status doesn't make sense here.

Reply:
ok